flocx_market/matcher.py

```diff
@@ -1,16 +1,17 @@
 import jmespath
 import re
+from flocx_market.api.offer import OfferList
 
 
 def apply_operator(val1, val2, op):
 
-    if val1 is None or val2 is None or op is None:
-        return False
-
     # null operator
-    if op == 'null':
+    if op is None and val1 is not None:
         return val1
 
+    if val1 is None or val2 is None:
+        raise ValueError
+
```

Comment:
I'm not sure we should raise a ValueError in this case.  Maybe someone legitimately wants to check if a list contains a null value?  `["somelist", "contains", None]`, etc.